src/components/atoms/OakCloudinaryImage/OakCloudinaryImage.test.tsx

```diff
@@ -34,7 +34,7 @@ describe(OakCloudinaryImage, () => {
     );
```

Comment:
Tests in this file were failing for some reason, amended them to pass but unsure as to the reason why they started to fail as not touched the compoent

Reply:
That is weird. `toMatch` should be matching on a substring. If anything the change should be more likely to break it 😭 🤯